### DISCUSSION TITLE
Unable to add a custom domain, no add domain button

### DISCUSSION THREAD

**Discussion by Touseef-ahmad** [community/community#147994] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In the documentation [Verifying a domain for your user site](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages#verifying-a-domain-for-your-user-site)

Following steps are mentioned

In the upper-right corner of any page on GitHub, click your profile photo, then click  Settings.

In the "Code, planning, and automation" section of the sidebar, click  Pages.

On the right, click Add a domain.

But I don't see add domain button see screenshot below

![Screenshot 2024-12-27 at 9 53 22 AM](https://github.com/user-attachments/assets/f5d7990d-34c8-4953-963f-fa40a2c5d293)

**Comment by lemorage** [comment, marked as answer]
Well, it looks like you’re starting off on the wrong page. You’re clicking on the repo settings page instead of the user settings page.

Here’s the correct way to do it:
1. Head over to your profile page at https://github.com/Touseef-ahmad.
2. Click on your profile photo in the top-right corner.
3. Select Settings from the dropdown.

After that, follow the same steps as outlined in the docs—you’ll find the `Add a domain` button there.

**Reply by Touseef-ahmad** [reply]
Thank you so much for clarifying that! I was indeed looking in the wrong place. Your instructions were helpful, and I’ve now found the correct settings. Appreciate your guidance!

**Comment by mecodeatlas** [comment]
Hi @Touseef-ahmad 👋🏾,It looks like @lemorage's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Reply by Touseef-ahmad** [reply]
Thank you, i have marked it as an answer!

**Reply by stoatsnhoney** [reply]
Thank you so much, I've been struggling with this for days
